### DISCUSSION TITLE
How to allow files to be written to a temporary folder?

### DISCUSSION THREAD

**Discussion by AButsai** [deta/space-docs#36] [Q&A] [answered]
Hello. Question. How to allow files to be written to the temporary folder tmp. the file is first uploaded to temp folder, then uploaded to firebase storage, after that the tmp folder is automatically cleared. How to allow writing files from deta.space side.

{"message":"EROFS: read-only filesystem, open '/var/task/src/tmp/f6689ddd-6879-4731-bb0d-50269bfb6343_4.jpg'"}

Many thanks for the help)

**Comment by abdelhai** [comment, marked as answer]
write to `/tmp`

**Reply by AButsai** [reply]
Thank you!
